SubmittedAssignments/Machhindranath_Kolape/preview.html

```diff
@@ -0,0 +1,281 @@
+                <div class="bottom-right-innerbox"></div>
+            </div>
+            <div class="left-gray-line1">
+                <div class="left-gray-line2"></div>
+            </div>
+            <div class="bottom-gray-line1">
+                <div class="bottom-gray-line2"></div>
+            </div>
+            <div class="left-dotted-line1">.....
+                <div class="left-dotted-line2">.....</div>
+            </div>
+            <div class="bottom-dotted-line1">.....
+                <div class="bottom-dotted-line2">.....</div>
+            </div>
+            <div class="image-borderbox"></div>
+            <div class="background-image">
+                <img src="img/background.jpg" alt="Background">
+            </div>
+            <div class="image">
+                <img src="img/background-1.jpeg" alt="Background-1">
+            </div>
+        </div>
+        <div class="content-wrapper">
+            <div class="org-logo">
+                <!-- Bootstrap icon -->
+                <img src="img/org-logo.png" alt="Logo">
+            </div>
+            <div class="org-tagline"></div>
+            <div class="org-address">
+                <!-- Bootstrap icon -->
+                <!-- <i class="fas fa-map-marker-alt"></i> -->
+                <p>Office No. 404, Vastu Viva, Beside Sanket Inn, Bhumkar Chowk
+                    Wakad, Pune, Maharashtra - 411057</p>
+            </div>
+            <div class="org-alternate-email">
+                <!-- Bootstrap icon -->
+                <!-- <i class="fas fa-envelope"></i> -->
+            </div>
+            <div class="org-phone">
+                <!-- Bootstrap icon -->
+                <!--<i class="fas fa-whatsapp"></i>-->
+                <div class="org-phone1">
+                    <img src="img/org-whatapp.png" alt="Logo" class="org-phone1">
+                </div>
+                +91 8999417889,
+            </div>
+            <div class="org-alternate-phone">
+                <!-- Bootstrap icon -->
+                <i class="fas fa-phone"></i>
+            </div>
+            <div class="org-fax">
+                <!-- Bootstrap icon -->
+                <i class="fas fa-fax"></i>
+            </div>
+            <div class="heading-first">Learn,Explore,Grow</div>
+            <div class="heading"> AI-Driven Learning Hub!
+            </div>
+            <div class="sub-heading"><b>DIGITAL STUDY CENTER</b>
+            </div>
+            <div class="description"> <p>Welcome to <b>SwayamGuru TechLok</b>, a premier study center powered by <b>Maha
+                Mission
+                Education and Career Council</b>-an avant-garde NGO committed to sculpting bright futures. Our
+                mission
+                is to create a dynamic learning environment where individuals can thrive, explore, and grow in the
+                realm of cutting-edge technology.</p>
+            </div>
+            <!--<div class="punchline-1">
```

Comment:
do not comment out the classes

Reply:
but I replace the element with the commented I-con dives provided by sample code only.  I didn't change the classes